besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -655,13 +655,12 @@ public Runner build() {
       final GraphQLDataFetchers fetchers =
           new GraphQLDataFetchers(
               supportedCapabilities, privacyParameters.getGoQuorumPrivacyParameters());
-      final GraphQLDataFetcherContextImpl dataFetcherContext =
-          new GraphQLDataFetcherContextImpl(
-              blockchainQueries,
-              protocolSchedule,
-              transactionPool,
-              miningCoordinator,
-              synchronizer);
+      final Map<String, Object> graphQlContextMap = new ConcurrentHashMap<>();
+      graphQlContextMap.putIfAbsent("blockchainQueries", blockchainQueries);
+      graphQlContextMap.putIfAbsent("protocolSchedule", protocolSchedule);
+      graphQlContextMap.putIfAbsent("transactionPool", transactionPool);
+      graphQlContextMap.putIfAbsent("miningCoordinator", miningCoordinator);
+      graphQlContextMap.putIfAbsent("synchronizer", synchronizer);
```

Comment:
looks fine overall. One change I would suggest is to make these strings constants eg BLOCKCHAIN_QUERIES. Maybe a new enum in the graphql package eg see GraphQLResponseType.